lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless government_data_availible?
+
```

Comment:
It feels a bit unusual to have new lines between the guard clauses.

Reply:
yep, sorry, had to leave one of them in because the linter complained though